ca_municipalities/people.py

```diff
@@ -0,0 +1,162 @@
+        else:
+            allowedClassifications = ['legislature', 'executive', 'upper', 'lower', 'party', 'committee', 'commission', 'corporation', 'agency', 'department', 'government']
+            if (organizationClassification not in allowedClassifications):
+                organizationClassification = 'government'
+
+        return organizationClassification
+
+    def scrape(self):
+        seat_numbers = defaultdict(lambda: defaultdict(int))
+
+        reader = self.csv_reader(self.csv_url, delimiter=self.delimiter, header=True, encoding=self.encoding, skip_rows=self.skip_rows)
+        reader.fieldnames = [self.header_converter(field) for field in reader.fieldnames]
+        for row in reader:
+
+            try:
+                if self.is_valid_row(row):
+                    for key, corrections in self.corrections.items():
+                        if not isinstance(corrections, dict):
+                            row[key] = corrections(row[key])
+                        elif row[key] in corrections:
+                            row[key] = corrections[row[key]]
+
+                    organizationClassification = row.get('classification')
+                    organizationClassification = self.validateOrganizationClassification(organizationClassification)
+
+                    organization = None
+                    organizationName = row['organization']
+                    if organizationName:
+                        if self.organizations.get(organizationName.lower()):
+                            organization = self.organizations.get(organizationName.lower())
+                        else:
+                            organization = Organization(organizationName, classification=organizationClassification)
+                            organization.add_source(self.csv_url)
+                            yield organization
+                            self.organizations[organizationName] = organization
+                    else:
+                        continue
+
+                    # ca_qc_laval: "maire et president du comite executif", "conseiller et membre du comite executif"
+                    # ca_qc_montreal: "Conseiller de la ville; Membre…", "Maire d'arrondissement\nMembre…"
+                    if row.get('primary role'):
+                        row['primary role'] = re.split(r'(?: (?:et)\b|[;\n])', row['primary role'], 1)[0].strip()
+                    else:
+                        row['primary role'] = 'Councillor'
+
+                    roleName = row.get('primary role')
+
+                    post = Post(role=roleName, label=organizationName, organization_id=organization._id)
+                    yield post
+
+                    if row.get('full name') and not re.search(r'[a-z]', row['full name']):
+                        row['full name'] = re.sub(r'(?<=\b[A-Z])[A-ZÀÈÉ]+\b', lambda x: x.group(0).lower(), row['full name'])
+
+                    name = '{}'.format(row['full name']).strip(' .,')
```

Comment:
No need to use format string. Just do `row['full name'].strip(' .,')` (There's punctuation at the start/end of names in the spreadsheet?)

Reply:
Yep